start-all.sh

```diff
@@ -216,6 +217,9 @@ for arg; do
 		--no-alertmanager)
 			SKIP_ALERTMANAGER=1
 			;;
+        --quick-startup)
+            QUICK_STARTUP=1
+            ;;
```

Comment:
Something looks off in the indentation here, but maybe it's just github showing it wrongly?